CMakeLists.txt

```diff
@@ -110,6 +110,10 @@ include(scripts/cmake/kconfig.cmake)
 add_dependencies(sof_options genconfig check_version_h)
 target_include_directories(sof_options INTERFACE ${GENERATED_DIRECTORY}/include)
 
+if(CONFIG_CAVS)
```

Comment:
This should go to  `src/platform/intel/cavs/CMakeLists.txt`, then you don't have even to guard it with `if`

Reply:
Ok, I've also add it to `sof/test/cmocka/src/lib/alloc/CMakeList.txt'